flasgger/base.py

```diff
@@ -309,7 +311,7 @@ def get_apispecs(self, endpoint='apispec_1'):
                 break
         if not spec:
             raise RuntimeError(
-                'Can`t find specs by endpoint {:d},'
+                'Can`t find specs by endpoint {},'
```

Comment:
When passing a non existing endpoint, this was breaking and throwing:

```bash
ValueError: Unknown format code 'd' for object of type 'str'
```

instead of the `RuntimeError`.